client/src/components/reviews/RatingBreakdown.jsx

```diff
@@ -48,11 +52,10 @@ const RatingBreakdown = ({data}) => {
               </div>
             );
           })}
-          <Recommended recommendedData={ratingOverview.recommended}/>
           <ProductBreakdown data={data}/>
         </div>
 
-        <div className="review-list">
+        <div className="review-list-container">
           <Sorting data={data} ratingFilter={ratingFilter}/>
```

Comment:
The <RatingFilterDesc />, <ProductBreakdown />, and  <Sorting /> components are being passed the same data, but you are breaking it  down slightly differently in RatingFilterDesc.  Is that to avoid a property name that is the similar to the component name?

Reply:
ProductBreakdown, Sorting and one other component I was referring to, I do not know why it removed them from the comment

Reply:
Yeah, the ratingFilter is locally produced, whereas the data was passed down from the parent component. The component names were my way to differentiate between the function it served. 

Rating Breakdown for rating specific (e.g. 1, 2, 3, 4, 5) 
Product Breakdown for product specific (e.g. length rating) 

And the Sorting component handles the logic to sort based on certain criteria (i.e. rating selected, newest, helpful etc.)